### ISSUE TITLE
Missing Features CodeWhisperer

### ISSUE THREAD

**Issue by jlong5795** [withfig/fig#2941]
I just installed CW today from the CLI and it appears that some of the features of Fig are missing and I wanted to confirm that to be the case or to see if I am missing something. For example, I had scripts installed (I used the commit script often) but I don't see a place to select or create such scripts. Is that a known fact, or am I missing something?

**Comment by SupianIDz** [comment]
Yes, scripts, dotfiles, plugins and servers are removed. You need to export your fig before upgrading it.

**Comment by HammyHavoc** [comment]
> Yes, scripts, dotfiles, plugins and servers are removed. You need to export your fig before upgrading it.

Not a Fig user, just passing through, but looked like a cool way to fiddle with dotfiles. Anything else you'd recommend in its place?

**Comment by filipef101** [comment]
Just to say I have found no alternative to fig GUI management of dotfiles, aliases and paths are the most relevant I believe, plugins are another story :/
